externs/custom-elements.js

```diff
@@ -12,6 +12,12 @@
 /** @type {boolean|undefined} */
 CustomElementRegistry.prototype.forcePolyfill;
 
+/** @type {boolean|undefined} */
+CustomElementRegistry.prototype.noConstructionObserver;
```

Comment:
We should expand this name to `noDocumentConstructionObserver`, I think the 'Document' part of it is pretty important.

Reply:
Done.